gadget/install/partition.go

```diff
@@ -70,21 +70,21 @@ type CreateOptions struct {
 // OnDiskStructure, as it is meant to be used externally (i.e. by
 // muinstaller).
 func CreateMissingPartitions(dl *gadget.OnDiskVolume, pv *gadget.LaidOutVolume, opts *CreateOptions) ([]gadget.OnDiskStructure, error) {
```

Comment:
wondering, not really related to the PR change as such: this exists only for muinstaller, I wonder if we shouldn't export createMissingPartitions as is instead, and let muinstaller deal with it?

Reply:
Might make sense to revisit this when we do the next step of building LaidOutStructures only when we have all data (we would use that structure in less places also).